.vscode/settings.json

```diff
@@ -9,20 +8,11 @@
         "editor.defaultFormatter": "esbenp.prettier-vscode",
         "editor.formatOnSave": true
     },
-
     "eslint.alwaysShowStatus": true,
-    "eslint.autoFixOnSave": true,
```

Comment:
Is this deprecated?

Reply:
![Screen Shot 2020-01-29 at 5 45 45 PM](https://user-images.githubusercontent.com/4524823/73404299-414d6780-42bf-11ea-90ac-b7fcaa7b5dd7.png)